src/gecondest.cc

```diff
@@ -95,28 +90,25 @@ void gecondest(
     scalar_t alpha = 1.;
     real_t Ainvnorm = 0.0;
 
-    GridOrder grid_order;
-    A.gridinfo(&grid_order, &p, &q, &myrow, &mycol);
-    slate_mpi_call(
-            MPI_Comm_size(A.mpiComm(), &mpi_size));
-    //myrow = A.mpiRank();
-    int64_t mloc  = numberLocalRowOrCol(m, nb, myrow, izero, p);
-    int64_t lldA  = blas::max(1, mloc);
-
-    std::vector<scalar_t> X_data(lldA);
-    std::vector<scalar_t> V_data(lldA);
-    std::vector<int64_t> isgn_data(lldA);
-    std::vector<int64_t> isave(3);
+    std::vector<int64_t> isave(4);
     isave[0] = 0;
     isave[1] = 0;
     isave[2] = 0;
-
-    auto X = slate::Matrix<scalar_t>::fromScaLAPACK(
-            m, 1, &X_data[0], lldA, nb, 1, p, q, A.mpiComm() );
-    auto V = slate::Matrix<scalar_t>::fromScaLAPACK(
-            m, 1, &V_data[0], lldA, nb, 1, p, q, A.mpiComm() );
-    auto isgn = slate::Matrix<int64_t>::fromScaLAPACK(
-            m, 1, &isgn_data[0], lldA, nb, 1, p, q, A.mpiComm() );
+    isave[3] = 0;
```

Comment:
`std::vector<int64_t> isave( 4 );` or more explicitly `isave( 4, 0 );` will actually already initialize all elements to 0 ([see docs](https://en.cppreference.com/w/cpp/container/vector/vector)).
Ok to change or leave as is, if desired for clarity.

Reply:
Also
```
std::vector<int64_t> x = { 0, 0, 0, 0 };
```
would be explicit.